### DISCUSSION TITLE
Binding für Datenpunkt

### DISCUSSION THREAD

**Discussion by SebastianSchultz** [Zefau/ioBroker.jarvis#1164] [Q&A] [answered]
Ist es möglich mittels Binding o.ä. zwei Datenpunkte für ein Gerät zu nutzen?

Hintergrund ist das ich z.B. im Esszimmer zwei Fenstersensoren habe, aber nun nicht für jeden Sensor ein Gerät anlegen möchte sondern z.B. nur ein Gerät "Esszimmer", welches bei true einer der beiden Sensoren auf "geöffnet" steht und bei false beider Sensoren auf "geschlossen".
Möchte wirklich sehr ungern den Umweg über ein Script oder ähnliches gehen , in VIS ging das ja auch.

**Comment by mcuiobroker** [comment, marked as answer]
https://mcuiobroker.gitbook.io/jarvis-infos/jarvis/besonderheiten-v3/binding/fenster-mit-2-haelften
Ich hoffe das Beispiel reicht aus, wenn nicht bitte eine Info geben.

**Comment by mcuiobroker** [comment]
@Zefau Ist beim Binding auch die Abfrage vom Zeitpunkt des anderen Wertes möglich?
`{Binding-DP}.ts<now()-1day?`

**Reply by Zefau** [reply]
Nein, musst du über einen eigenen Datenpunkt in ioBroker lösen. Es wird immer auf den Wert aufgelöst.
